dawn/src/dawn/CodeGen/CXXUtil.h

```diff
@@ -661,10 +661,10 @@ struct Struct : public Structure {
       : Structure("struct", name, s, templateName) {}
 };
 
-auto c_gt = []() { return std::string("gridtools::"); };
-auto c_dgt = []() { return std::string("gridtools::dawn::"); };
-auto c_gt_enum = []() { return std::string("gridtools::enumtype::"); };
-auto c_gt_intent = []() { return std::string("gridtools::intent::"); };
+const std::string c_gt = "gridtools::";
```

Comment:
I like the idea of simplifying this, but won't this potentially cause duplicate symbols? Would declaring here and defining in the source be an option?

Reply:
There's is no source/cpp file for `CXXUtil.h`. Without `const` it leads to duplicate symbols (-> linker error -> multiple definitions).

We're not sure whether adding a cpp file and adjusting the cmake files makes sense for this small change.

Reply:
I looked at this: https://en.cppreference.com/w/cpp/language/storage_duration
And `const` doesn't necessarily implie `static`, but `static` should still be redundant in this case.

`static` implies:

> static - static or thread storage duration and internal linkage. 

`const` implies internal linkage:

> Any of the following names declared at namespace scope have internal linkage:
> ...
>  non-volatile non-template (since C++14) non-inline (since C++17) non-exported (since C++20) **const-qualified variables** (including constexpr) that aren't declared extern and aren't previously declared to have external linkage;
> ...

Compiling with `static const` succeeds with gcc 10.1